<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="chooseOpponentLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>501</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose your opponent</string>
    </property>
   </widget>
   <widget class="QPushButton" name="computerButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Computer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playerButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>350</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
